File: rest/src/main/java/io/sparkled/rest/service/effecttype/EffectTypeRestServiceHandler.java
package io.sparkled.rest.service.effecttype;

import io.sparkled.model.animation.effect.reference.EffectTypes;
import io.sparkled.rest.service.RestServiceHandler;

import javax.ws.rs.core.Response;

public class EffectTypeRestServiceHandler extends RestServiceHandler {

    public Response get() {
        return respondOk(EffectTypes.get());
    }
}

File: rest/src/main/kotlin/io/sparkled/rest/service/effecttype/EffectTypeRestServiceHandler.kt
package io.sparkled.rest.service.effecttype

import io.sparkled.model.animation.effect.reference.EffectTypes
import io.sparkled.rest.service.RestServiceHandler

import javax.ws.rs.core.Response

open class EffectTypeRestServiceHandler : RestServiceHandler() {

    fun get(): Response {
        return respondOk(EffectTypes.get())
    }
}